src/lib/export/formats/csv.ts

```diff
@@ -0,0 +1,39 @@
+import { Export } from "@/lib/export/export";
+import { DBConnection, TableOrView, TableFilter } from '@/lib/db/client'
+
+interface OutputOptionsCsv {
+  header: boolean,
+  delimiter: string
+}
+
+export default class CsvExporter extends Export {
+  readonly format: string = 'csv'
+
+  constructor(
```

Comment:
I don't have much TypeScript experience and I didn't find a way to avoid having to declare a constructor override only for the format-specific `OutputOption` interface. I tried declaring a generic `OutputOptions` interface in the `export.ts` and extending that, but it doesn't work that way apparently.

Reply:
I'll fix this, don't worry :-)